internal/integration/telemetry_sdk_emitter.go

```diff
@@ -42,6 +42,10 @@ type TelemetryEmitterConfig struct {
 	// DeltaExpirationCheckInternval sets the cumulative DeltaCalculator
 	// duration between checking for expirations. Defaults to 30s.
 	DeltaExpirationCheckInternval time.Duration
+
+	// boundedHarvester configuration
+	DisableBoundedHarvester bool
```

Comment:
is this for future work?
I do not see where it can be configured, currently is is "hardcoded" to false?

Reply:
I put this for the integration tests for the telemetry_sdk, as since they were not thought with the BoundedHarvester in mind, they broke in unexpected ways.

That being said, I added this option before the ability to stop the periodic harvest routine was implemented, so perhaps the tests will work nice now just with that.

I'll give it a try.

Reply:
It seems that `DisablePeriodicReporting` is indeed enough and `DisableBoundedHarvester` is no longer required for the integration sdk emmiter tests to pass.

However, due to the asynchronous nature of `boundedHarvester`, this flag is still useful for `BenchmarkTelemetrySDKEmitter`.

I'd suggest to keep the flag for now, at least until performant reporting is merged to the infra-sdk and we can get rid of the asynchronicity of `boundedHarvester.HarvestNow`